topics/ecology/tutorials/gbif_cleaning/tutorial.md

```diff
@@ -0,0 +1,367 @@
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+{: .hands_on}
+
+
+## Where come from the records?
+
+> <hands-on-title>Here we propose to investigate the content of the dataet looking notably at the "basisOfRecord" attribute to know more about heterogenity related to the data collection origin.</hands-on-title>
+>
+> 1. **Count** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"from dataset"*: `output` (output of **Get species occurrences data** {% icon tool %})
+>    - *"Count occurrences of values in column(s)"*: `c[17]`
+>
+>
+>    > <comment-title></comment-title>
+>    >
+>    > This tool is one of the important "classical" Galaxy tool who allows you to better synthtize information content of your data. Here we apply this tool to the 17th column (corresponding to the basisOfRecord attribute) but don't hesitate to investigate others attributes!
+>    {: .comment}
+>
+{: .hands_on}
+
+> <question-title></question-title>
+>
+> 1. How many different types of data collection origin are there?
+> 2. What is your assumption regarding this heterogeneity?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. 5
+> > 2. each basisOfRecord type is related to different collection method so different data quality
+> >
+> {: .solution}
+>
+{: .question}
+
+## Filtering data thanks to the data origin
+
+> <hands-on-title>Filter data on basisOfRecord GBIF attribute</hands-on-title>
+>
+> 1. **Filter** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Filter"*: `output` (output of **Get species occurrences data** {% icon tool %})
+>    - *"With following condition"*: `c17=='HUMAN_OBSERVATION' or c17=='OBSERVATION' or c17=='PRESERVED_SPECIMEN'`
+>    - *"Number of header lines to skip"*: `1`
+>
+>    > <comment-title></comment-title>
+>    >
+>    > A comment about the tool or something else. This box can also be in the main text
+>    {: .comment}
+>
+>
+>    > <question-title></question-title>
+>    >
+>    > 1. How many records are kept and what is the percentage of filtered data?
+>    > 2. Why are we keeping only these 3 types of data collection origin?
+>    >
+>    > > <solution-title></solution-title>
+>    > >
+>    > > 1. 470 and 8.79% of records were drop out
+>    > > 2. These data collection methods are the most relevant
+>    > >
+>    > {: .solution}
+>    >
+>    {: .question}
+>
+> 3. Add to the output dataset a propagating tag corresponding to the filtering criteria adding `#basisOfRecord` string for example
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+{: .hands_on}
+
+
+## Have a look at the number of counts per record
+
+> <hands-on-title>Here we propose to have a look at the number of counts by record to know if there is some possible records with errors.</hands-on-title>
```

Comment:
another full sentence as box title. Move the sentence above the box and enter a short box title

Reply:
THANK YOU!